### PR TITLE
PLAT-126331: Add UI tests for agate/RadioItem

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - Added ui tests for agate/RadioItem

### Links
[//]: # (Related issues, references)
PLAT-126331

### Comments

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: tests/ui/specs/RadioItem/Agate-RadioItem-Page.js [hunk 1]
```diff
@@ -20,6 +20,9 @@ class AgateRadioItemInterface {
 	get isSelected () {
```
Comment (jeonghee27) [inline tests/ui/specs/RadioItem/Agate-RadioItem-Page.js:20]: You don't need to use'-' for Page file name.
In the PLAT-126330, there was not.
Agate-PanelPage.js is Better.
In fact, I don't think it's good to prefix of "Agate-".
"-Spec" and "-View" is mandatory to filter on ui-test-utils. but Page is not. just use camelcase.

**Review by jeonghee27 — APPROVED** [review]
LGTM except file name

File: tests/ui/specs/RadioItem/RadioItemPage.js [hunk 2]
```diff
@@ -20,21 +20,28 @@ class AgateRadioItemInterface {
...
-class AgateRadioItemPage extends Page {
+class RadioItemPage extends Page {
 	constructor () {
 		super();
-		this.title = 'Agate RadioItem Test';
+		this.title = 'RadioItem Test';
 		const radioDefault = new AgateRadioItemInterface('radioItem1');
 		const radioDefaultSelected = new AgateRadioItemInterface('radioItem2');
-		this.components = {radioDefault, radioDefaultSelected};
+		const radioInline = new AgateRadioItemInterface('radioItem3');
```
Comment (MikyungKim) [inline tests/ui/specs/RadioItem/RadioItemPage.js:35]: Would you please remove `Agate` from `RadioItemInterfae`?

File: tests/ui/specs/RadioItem/RadioItem-specs.js [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+		it('should have correct text', function () {
+			expect(radioItem.textContent).to.equal('Radio Item inline disabled');
+		});
+
+		it('should be selected', function () {
+			expect(radioItem.isSelected).to.be.true();
+		});
+
+		it('should display inline item', function () {
```
Comment (MikyungKim) [inline tests/ui/specs/RadioItem/RadioItem-specs.js:192]:
```suggestion
		it('should display item inline', function () {
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**